src/content/topics/integrations/observability/chronosphere.mdx

```diff
@@ -0,0 +1,74 @@
+---
+path: /integrations/chronosphere
+title: Chronosphere
+description: This topic explains how to configure and send LaunchDarkly events to
+Chronosphere.
+published: true
+tags: ['chronosphere', 'integration', 'event', 'observability']
+---
+
+## Overview
+
+This topic explains how to configure and send LaunchDarkly feature flag events to
+[Chronosphere](https://chronosphere.io/). These events display as change events
+within the Chronosphere Changes Explorer, and as change events on dashboards and
+query results for monitors. This integration lets you visualize more of your
+LaunchDarkly events within Chronosphere and have a single location for identifying
+anomalies across your environment.
+
+## Prerequisites
+
+To configure the Chronosphere integration, you need the following information:
+
+- **Chronosphere receiver URL**: This URL is the destination where LaunchDarkly
+  sends data to Chronosphere. The URL is structured as your Chronosphere tenant
+  URL, prefixed to the LaunchDarkly receiver endpoint. For example:
+  `https://MY_COMPANY.chronosphere.io/api/v1/data/events/receiver/launchdarkly`.
+- **Chronosphere secret token**: This credential is an API token you obtain from
+  Chronosphere Support to authenticate with Chronosphere.
+
+## Configuring the integration
+
+To configure the Chronosphere integration:
+
+1. Navigate to the [Integrations page](https://app.launchdarkly.com/default/integrations) page and find "Chronosphere".
+1. Click **Add integration**. The "Create Chronosphere configuration" panel appears.
+1. (Optional) Enter a human-readable **Name**, such as **Chronosphere**.
```

Comment:
```suggestion
1. (Optional) Enter a human-readable **Name**, such as "Chronosphere".
```
nit, we use bolding for field names and clickable elements, but not for user-entered text.